packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py

```diff
@@ -17,7 +17,11 @@
 
 # To avoid any side effects using the same timeout as the
 # aiohttp client that was calling into director-v2 from webserver
-_DEFAULT_LEGACY_TIMEOUT_S: Final[NonNegativeInt] = 20
+DEFAULT_LEGACY_TIMEOUT_S: Final[NonNegativeInt] = 20
+
+# make sure RPC calls time out after the HTTP requests
+# from dynamic-scheduler to director-v2 time out
+_RPC_DEFAULT_TIMEOUT_S: Final[NonNegativeInt] = 20 + 1
```

Comment:
1 second? is that really enough? I don't want to play the sleep time thing here. but usually times like these tend to smell bad... and have various constraints depending on the machines where they run etc etc... I thought the idea was to go with order of magnitudes, or maybe double the first one... well your call here.